test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -55,19 +56,22 @@ class AndroidArgs(
     private val androidGcloud = androidGcloudYml.gcloud
     var appApk = (cli?.app ?: androidGcloud.app ?: throw FlankFatalError("app is not set")).processFilePath("from app")
     var testApk = (cli?.test ?: androidGcloud.test)?.processFilePath("from test")
-    val additionalApks = (cli?.additionalApks ?: androidGcloud.additionalApks).map { it.processFilePath("from additional-apks") }
+    val additionalApks =
+        (cli?.additionalApks ?: androidGcloud.additionalApks).map { it.processFilePath("from additional-apks") }
     val autoGoogleLogin = cli?.autoGoogleLogin ?: cli?.noAutoGoogleLogin?.not() ?: androidGcloud.autoGoogleLogin
 
     // We use not() on noUseOrchestrator because if the flag is on, useOrchestrator needs to be false
     val useOrchestrator = cli?.useOrchestrator ?: cli?.noUseOrchestrator?.not() ?: androidGcloud.useOrchestrator
-    val roboDirectives = cli?.roboDirectives?.parseRoboDirectives() ?: androidGcloud.roboDirectives.parseRoboDirectives()
+    val roboDirectives =
+        cli?.roboDirectives?.parseRoboDirectives() ?: androidGcloud.roboDirectives.parseRoboDirectives()
```

Comment:
Is there a reason why the formatting is being changed here?

Reply:
It's auto code format change. Before save file I always use auto code format. If it wrong or I shouldn't commit this changes let me know, I can revert it. :)

Reply:
Auto wrapping isn't works well in many cases i prefer to do it by hand. Its also good idea to expand setting `General -> Code Style -> Hard wrap at` to 200 for example, to avoid accidental wrappings.

Reply:
Thanks! Now i set Hard wrap to 200 i revert changes.